newsfragments/10342.fixed

```diff
@@ -0,0 +1 @@
+Catches and ignores errors during the directory fetch for ARI checking so that these errors do not hinder the actual certificate issuance. Certbot also now uses the server value from the renewal configuration file for ARI checks instead of the server value from the current invocation of Certbot. This helps prevent ARI requests from going to the wrong server if the user changes CAs.
```

Comment:
any reason not to keep these two changelog entries separate, deleting all but the first sentence here, letting it link to #10342, and creating a new 10339.fixed file for the

> Certbot now always uses the server value from the renewal configuration file for ARI checks instead of the server value from the current invocation of Certbot. This helps prevent ARI requests from going to the wrong server if the user changes CAs.

fix for #10339?

i also noticed this moved the first sentence here from "changed" to "fixed". if you meant to do this, that's fine with me, otherwise, i'd suggest renaming this file 10342.changed